eng/Version.Details.xml

```diff
@@ -93,6 +93,10 @@
       <Uri>https://github.com/dotnet/wpf</Uri>
       <Sha>8c5494aa7298e8182ccb4639eb52b002458d0158</Sha>
     </Dependency>
+    <Dependency Name="System.ServiceModel.Primitives" Version="4.7.0-preview1.final">
```

Comment:
Comment here why we're pulling in a package we don't use.

Reply:
Comment added.